factory.py

```diff
@@ -149,19 +149,18 @@ def main():
     FACTORY_DB = json.loads(read_file("factory.json"))
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--single", nargs=2, type=str)
-    parser.add_argument("--all", action="store_true")
-    parser.add_argument("--create-prs", action="store_true")
+    parser.add_argument("--single", nargs=2, type=str, metavar=('<shortname>', '<h1>'), help="generate a single standard, e.g., --single xhr XMLHttpRequest")
```

Comment:
```suggestion
    parser.add_argument("--single", nargs=2, type=str, metavar=("<shortname>", "<h1>"), help="generate a single standard, e.g., --single xhr XMLHttpRequest")
```

Seems strange to use double-quotes for all other nearby strings and single-quotes here.